spellbook/models/seaport/ethereum/seaport_ethereum_transfers.sql

```diff
@@ -0,0 +1,795 @@
+-- ## this function consists of 4 parts,
+-- ## p1_ : seaport."Seaport_call_fulfillBasicOrder"
+-- ## p2_ : seaport."Seaport_call_fulfillAvailableAdvancedOrders"
+-- ##       seaport."Seaport_call_fulfillAvailableOrders"
+-- ## p3_ : seaport."Seaport_call_fulfillOrder"
+-- ##       seaport."Seaport_call_fulfillAdvancedOrder"
+-- ## p4_ : seaport."Seaport_call_matchOrders"
+-- ##     : seaport."Seaport_call_matchAdvancedOrders"
+
+{{ config(
+        alias ='transfers',
+        materialized ='incremental',
+        file_format ='delta',
+        incremental_strategy='merge',
+        unique_key='unique_trade_id'
+        )
+}}
+
+with p1_call as (
+    select 'basic_order' as main_type
+          ,call_tx_hash as tx_hash
+          ,call_block_time as block_time
+          ,call_block_number as block_number
+          ,max(get_json_object(parameters, "$.basicOrderType")) as order_type_id
+      from seaport_ethereum.Seaport_call_fulfillBasicOrder
```

Comment:
Done thanks ! Added missing jinja {{ref()}} (just realized my commit for this didn't go through)

Reply:
i notice now that since this file reads from the source directly/references the source file, it uses {{source()}} syntax vs. ref, which appears to be used on models built rather than the source directly. makes sense, thanks!